src/main/java/thestonedturtle/partypanel/ui/PlayerPanel.java

```diff
@@ -192,4 +244,40 @@ public void changePlayer(final PartyPlayer newPlayer)
 			prayersPanel.updatePrayerRemaining(player.getSkillBoostedLevel(Skill.PRAYER), player.getSkillRealLevel(Skill.PRAYER));
 		}
 	}
+
+	private void updatePanel()
+	{
+		this.removeAll();
+
+		this.setBorder(new CompoundBorder(
+				new MatteBorder(2, 2, 2, 2, new Color(87, 80, 64)),
+				new EmptyBorder(0, 0, 5,  0)
+		));
+
+		final JPanel view = new JPanel();
+		view.setBorder(new EmptyBorder(5, 5, 0,  5));
+		final MaterialTabGroup tabGroup = new MaterialTabGroup(view);
+		tabGroup.setBorder(new EmptyBorder(10, 0, 4, 0));
+
+		addTab(tabGroup, SpriteID.TAB_INVENTORY, inventoryPanel, "Inventory");
+		addTab(tabGroup, SpriteID.TAB_EQUIPMENT, equipmentPanel, "Equipment");
+		addTab(tabGroup, SpriteID.TAB_PRAYER, prayersPanel, "Prayers");
+		addTab(tabGroup, SpriteID.TAB_STATS, skillsPanel, "Skills");
+
+		setLayout(new DynamicGridLayout(0, 1));
+
+		if (this.showInfo)
+		{
+			add(banner);
+			add(tabGroup);
+			add(view);
+		}
+		else
```

Comment:
Always add the banner above the if statement and then you can remove this else condition